directives/abstract_exercise.py

```diff
@@ -23,3 +23,21 @@ def extract_difficulty_and_points(self, arg):
                 else:
                     difficulty = u''.join(chars)
         return difficulty, points
+
+    def set_assistant_permissions(self, data):
+
+        if 'allow-assistant-grading' in self.options:
+            if str(self.options['allow-assistant-grading']).lower() == 'true':
+                data.update({'allow_assistant_grading': True})
```

Comment:
I forgot about the choice type in the option spec. Since this option has choices true or false, it is better to use the choice type instead of `directives.unchanged`. Then Sphinx/docutils will check if the user input matches any valid choice and you don't need this if statement.